joblib/test/test_numpy_pickle.py

```diff
@@ -333,14 +334,15 @@ def test_compress_mmap_mode_warning():
         numpy_pickle.load(this_filename, mmap_mode='r+')
         nose.tools.assert_equal(len(caught_warnings), 1)
         for warn in caught_warnings:
-            nose.tools.assert_equal(warn.category, DeprecationWarning)
+            nose.tools.assert_equal(warn.category, Warning)
```

Comment:
Because this is the type of warning raised [here](https://github.com/joblib/joblib/pull/351/files/6e9d3625fa75b2eff5a5afd9930c27f180a53903#r67527746) and that we want to test: it checks one cannot load a compressed pickle using mmap_mode so I think there's no meaning using a DeprecationWarning in this case.